# Node parameters
/**:
  ros__parameters:
    base_link_frame_id: "base_link"
    time_sync_threshold: 0.05
    sub_object_timeout_sec: 0.5

    # choose the input sensor type for each object type
    # "lidar", "radar", "camera" are available
    main_sensor_type: "lidar"
    sub_sensor_type: "radar"

    # tracker settings
    tracker_state_parameter:
      remove_probability_threshold: 0.3
      publish_probability_threshold: 0.5
      default_lidar_existence_probability: 0.8
      default_radar_existence_probability: 0.5
      default_camera_existence_probability: 0.6
      decay_rate: 0.1
      max_dt: 0.2
